pvlib/solarposition.py

```diff
@@ -15,50 +15,55 @@
 import numpy as np
 import pandas as pd
 import pytz
-    
+
 
 from pvlib.tools import localize_to_utc, datetime_to_djd, djd_to_datetime
 
 
-def get_solarposition(time, location, method='pyephem', pressure=101325, 
+def get_solarposition(time, location, method='pyephem', pressure=101325,
                       temperature=12):
     """
     A convenience wrapper for the solar position calculators.
-    
+
     Parameters
     ----------
     time : pandas.DatetimeIndex
     location : pvlib.Location object
     method : string
-        'pyephem' uses the PyEphem package. Default.
-        'spa' uses the pvlib ephemeris code.
-    pressure : float
+        'pyephem' uses the PyEphem package (default): :func:`pyephem`
+
+        'spa' uses the spa code: :func:`spa`
+
+        'ephemeris' uses the pvlib ephemeris code: :func:`ephemeris`
+    pressure : float [default: 101325]
         Pascals.
-    temperature : float 
+    temperature : float [default: 12]
```

Comment:
Most of the pvlib docstrings do not have defaults in the parameter descriptions. This is mostly because I personally don't like to put them there, and I removed them when cleaning up the docstrings. I think I might be in the minority, though, so maybe consider starting an issue for this. I'm ok with keeping this here, as long as we know about the inconsistencies.